.github/workflows/CI.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
       fail-fast: true
       matrix:
         os: [macOS-latest, ubuntu-latest]
-        python-version: ['3.8', '3.9', '3.10']
+        python-version: ['3.11']
```

Comment:
[psi4 conda packages exist](https://psicode.org/installs/v19/) for python 3.12, do we want to go ahead and add that to the testing matrix?

Reply:
Yes, I'll add that.

Reply:
Hmmm...it fails when trying to install pytorch:

`ERROR: Could not find a version that satisfies the requirement torch (from pycc) (from versions: none)
ERROR: No matching distribution found for torch`

Reply:
I don't this is going to happen right now. It appears that `pytorch` [isn't yet ready for python 3.12](https://github.com/pytorch/pytorch/issues/110436), so I'm going to stick with 3.11 at this stage.